ProtocolHandler.pm

```diff
@@ -1,87 +1,147 @@
 package Plugins::RadioParadise::ProtocolHandler;
 
 use strict;
-# XXX - https
-use base qw(Slim::Player::Protocols::HTTP);
+use base qw(IO::Handle);
 
 use JSON::XS::VersionOneAndTwo;
 use Tie::Cache::LRU;
 use POSIX qw(ceil);
-
-use Slim::Networking::SimpleAsyncHTTP;
+use List::Util qw(min max);
 
 use Slim::Utils::Log;
 use Slim::Utils::Prefs;
 use Slim::Utils::Timers;
+use Slim::Utils::Errno;
+use Slim::Networking::Async::HTTP;
 
-use Plugins::RadioParadise::Stations;
```

Comment:
We still need this, don't we?

Reply:
yes, crap - why doesn't it break one mine. I always forget in Perl is we need a use sequence when calling explicitely a package function like getStreamParams does. Anyway, I've added back and it It was not intentional anyway